stack_data/core.py

```diff
@@ -178,19 +206,17 @@ def render_with_markers(self, markers, strip_leading_indent=True):
         return ''.join(parts)
 
 
-Range = namedtuple('Range', 'start end data')
-
 
-def markers_from_ranges(ranges, converter):
+def markers_from_ranges(ranges:List[Range], converter)->List[Marker]:
```

Comment:
```suggestion
def markers_from_ranges(ranges: List[Range], converter) -> List[Marker]:
```

Please keep the code PEP8 compliant.